BUILD.bazel

```diff
@@ -158,12 +151,7 @@ filegroup(
 
 cc_binary(
     name = "protoc",
-<<<<<<< HEAD
-    linkopts = LINK_OPTS,
-=======
-    srcs = ["src/google/protobuf/compiler/main.cc"],
```

Comment:
`main.cc` is not visible from the top-level package, but it is already included in `protoc_lib`